webapp/src/types/model.ts

```diff
@@ -157,7 +157,6 @@ export type ChannelSubscription = {
 export enum InstanceType {
     CLOUD = 'cloud',
     SERVER = 'server',
-    CLOUD_OAUTH = 'cloud_oauth'
```

Comment:
This seems wrong to get rid of this. Where are the other values used on the frontend?

Reply:
Looking at the webapp code, I could only find the usage of `InstanceType` in the Connect Modal flow, i.e. `/jira connect`. However, even this works fine without the requirement of having the InstanceType `CLOUD_OAUTH = 'cloud_oauth'`. This was initially added to handle the slash command `/jira instance install cloud-oauth` via the webapp flow. However since this behavior was changed as per your review comment [here](https://github.com/mattermost/mattermost-plugin-jira/pull/928#discussion_r1156449925). Hence we no longer have any dependence or usage of having `CLOUD_OAUTH` in the InstanceType. Please let me know if I have overlooked something here?